README.md

```diff
@@ -0,0 +1,59 @@
+# nodejs-custom-plugin
```

Comment:
We should improve this readme adding an example usage and the required dependencies to be used with. Change the readme title with the name of the plugin, it is not the generic `nodejs-custom-plugin`